feature/search/src/main/kotlin/uk/govuk/app/search/SearchUiState.kt

```diff
@@ -0,0 +1,20 @@
+package uk.govuk.app.search
+
+import uk.govuk.app.search.data.remote.model.Result
+
+internal sealed class SearchUiState(
```

Comment:
I don't think we need these in here as they are only relevant in the default state?